src/main/scala/com/microsoft/sqlserver/jdbc/spark/utils/BulkCopyUtils.scala

```diff
@@ -160,6 +166,18 @@ object BulkCopyUtils extends Logging {
         conn.createStatement.executeQuery(queryStr)
     }
 
+    /**
+    * getComputedCols
+    * utility function to get computed columns.
+     * Use computed column names to exclude computed column when matching schema.
+    */
+    private[spark] def getComputedCols(
```

Comment:
getComputeCols should return a list of computed columns. That's more natural for better structure. Why would be leave ResultSet processing for external user. Thoughts?

Reply:
Yes agree. Now getComputeCols() returns a list of computed column names.